common/common_helper.sh

```diff
@@ -692,6 +692,36 @@ function kruize_local_ros_patch() {
   	fi
 }
 
+#
+# "kafka" flag is turned off by default for now. This needs to be turned on.
+#
+function kruize_kafka_patch() {
+  echo -n "Applying Kafka Patch..."
+	CRC_DIR="./manifests/crc/default-db-included-installation"
+	KRUIZE_CRC_DEPLOY_MANIFEST_OPENSHIFT="${CRC_DIR}/openshift/kruize-crc-openshift.yaml"
+	KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE="${CRC_DIR}/minikube/kruize-crc-minikube.yaml"
+	KRUIZE_CRC_DEPLOY_MANIFEST_AKS="${CRC_DIR}/aks/kruize-crc-aks.yaml"
+
+	pushd autotune >/dev/null
+		# Checkout mvp_demo to get the latest mvp_demo release version
+		git checkout mvp_demo >/dev/null 2>/dev/null
+
+		if [ ${CLUSTER_TYPE} == "kind" ]; then
+			sed -i 's/"isKafkaEnabled" : "false"/"isKafkaEnabled" : "true"/' ${KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE}
+			# Replace the existing KAFKA_BOOTSTRAP_SERVERS value
+      sed -i "s|value: \"kruize-kafka-cluster-kafka-bootstrap.kafka.svc.cluster.local:9092\"|value: \"$BOOTSTRAP_SERVER\"|g" ${KRUIZE_CRC_DEPLOY_MANIFEST_MINIKUBE}
```

Comment:
Trying the demo in multiple namespaces doesn't replace the actual bootstrap server